airbyte-workers/src/main/java/io/airbyte/workers/normalization/NormalizationAirbyteStreamFactory.java

```diff
@@ -93,10 +93,9 @@ private Stream<AirbyteMessage> filterOutAndHandleNonAirbyteMessageLines(JsonNode
         try (final var mdcScope = containerLogMdcBuilder.build()) {
           switch (logLevel) {
             case "debug" -> logger.debug(logMsg);
-            case "info" -> logger.info(logMsg);
             case "warn" -> logger.warn(logMsg);
             case "error" -> logAndCollectErrorMessage(logMsg);
-            default -> logger.info(jsonLine.asText()); // this shouldn't happen but logging it to avoid hiding unexpected lines.
+            default -> logger.info(logMsg); // this shouldn't happen but logging it to avoid hiding unexpected lines.
```

Comment:
this seems like it would render jsonLines differently than the current impl -- what's the upside to doing it this way?

Reply:
Did have some wider changes in this part and missed changing this bit back, reverted.